pkg/util/overridemanager/overridemanager.go

```diff
@@ -190,7 +190,14 @@ func (o *overrideManagerImpl) getOverridersFromOverridePolicies(policies []Gener
 			resourceMatchingPolicies = append(resourceMatchingPolicies, policy)
 		}
 	}
-
+	sort.SliceStable(resourceMatchingPolicies, func(i, j int) bool {
+		implicitPriorityI := util.ResourceMatchSelectorsPriority(resource, resourceMatchingPolicies[i].GetOverrideSpec().ResourceSelectors...)
+		implicitPriorityJ := util.ResourceMatchSelectorsPriority(resource, resourceMatchingPolicies[j].GetOverrideSpec().ResourceSelectors...)
+		if implicitPriorityI != implicitPriorityJ {
+			return implicitPriorityI < implicitPriorityJ
```

Comment:
I'm worried that the behavior of `op` is inconsistent with `pp`, which will confuse users.

We need more documents to explain

Reply:
Yes, would u mind to add some documents to explain? here is the example: https://github.com/karmada-io/website/blob/main/docs/userguide/scheduling/resource-propagating.md, chapter: `Configure PropagationPolicy priority
`

Reply:
ok ok

Reply:
> I'm worried that the behavior of op is inconsistent with pp, which will confuse users.

Indeed, the one with higher priority but applied after the one with lower priority.

Document it might not be the best choice, we should consider if the `priority` concept is appropriate with OP/COP.